server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -171,14 +176,30 @@ export class RwcdsFormService {
       const projectActionLabel = action['_dcp_action_value@OData.Community.Display.V1.FormattedValue'];
 
       // Lookup: dcp_affectedzoningresolutions asks for a ZR type,
-      const { label, code } = ZONING_RESOLUTION_TYPES.filter((zr) => zr.label === projectActionLabel)[0] || { label: '', code: null };
+      const { label, code } = ZONING_RESOLUTION_TYPES.find((zr) => zr.label === projectActionLabel) || { label: '', code: null };
+      const matchingZr = affectedZoningResolutions.find(zr => zr.dcp_zoningresolutiontype === code);
+      // action attribute that matches dcp_zrsectionnumber on dcp_affectedzoningresolution
+      const currentActionZoningResolution = action.dcp_ZoningResolution ? action.dcp_ZoningResolution.dcp_zoningresolution : null
 
       if (!label) console.log(`Could not find Affected ZR Type for ${projectActionLabel}`);
 
-      if (!zrTypes.includes(label) && label) {
+      // IF the action.ZoningResolution.dcp_zoningresolution value has changed since the initial POST,
+      // then PATCH the new value to dcp_zrsectionnumber on dcp_affectedzoningresolution entity
+      // NOTE: dcp_modifiedzrsectionnumber not included here because it's updated by applicant on rwcds-form
+      if (matchingZr && matchingZr.dcp_zrsectionnumber !== currentActionZoningResolution) {
+        return this.crmService.update('dcp_affectedzoningresolutions', matchingZr.dcp_affectedzoningresolutionid, {
+          dcp_zrsectionnumber: currentActionZoningResolution,
+        });
+      }
+
+      // IF the zr type does not exist yet, then POST the action
+      // NOTE: some actions that exist in dcp_projectaction SHOULD NOT be copied over
+      // NOTE cont. -- The ZONING_RESOLUTION_TYPES lookup also functions as a list of actions that SHOULD...
```

Comment:
tiny style nit: Not sure about these `NOTE cont.` comment prefixes 
I think it's fine to just continue the sentence on the subsequent line

Reply:
lol sounds good :)